Update docs (#73)
urlscan/urlscan-cli: README.md

The built binaries can be found on [the releases page](https://github.com/urlscan/urlscan-cli/releases).

> [!NOTE]

> > Apple's [Gatekeeper](https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web) may block the binary to run.

Reviewer: ```suggestion
> Apple's [Gatekeeper](https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web) may prevent the binary from running.
```

> > Please do `xattr -c urlscan` to unblock it. Or use Homebrew Cask (see the blow). The Homebrew Cask installed binary is Gatekeeper-free.

Reviewer: ```suggestion
> Use `xattr -c urlscan` to unblock it or install via Homebrew Cask (see below).
```

> > Please do `xattr -c urlscan` to unblock it. Or use Homebrew Cask (see the blow). The Homebrew Cask installed binary is Gatekeeper-free.

Reviewer 2: ```suggestion
> Please run `xattr -c urlscan` to allow execution, or install via Homebrew Cask (see blow).
```
